pkgs/applications/version-management/git/default.nix

```diff
@@ -37,7 +37,8 @@
   nlsSupport ? true,
   osxkeychainSupport ? stdenv.hostPlatform.isDarwin,
   guiSupport ? false,
-  withManual ? true,
+  # Disable the manual since libxslt doesn't seem to parse the files correctly.
+  withManual ? !stdenv.hostPlatform.useLLVM,
```

Comment:
Last minute thought: I initially wanted to merge this but then I asked myself how this interacts with darwin? Does it disable the manual there?

Reply:
Darwin doesn't get impacted since `useLLVM` is false on Darwin.